src/state/component-repository/components/reducer.js

```diff
@@ -202,7 +206,7 @@ const filters = (state = filtersDefaultState, action = {}) => {
         ? addOrUpdateFilter(filter, state, category)
         : removeFilter(filter, state, category);
     }
-    case CLEAR_DE_SEARCH_FILTER: {
+    case CLEAR_ECR_SEARCH_FILTER: {
       const category = action.payload.digitalExchangeCategory;
```

Comment:
I think this needs to be changed to: `action.payload.componentRepositoryCategory`